website/content/api-docs/system/config-group-policy-application.mdx

```diff
@@ -0,0 +1,82 @@
+---
+layout: api
+page_title: /sys/config/group-policy-application - HTTP API
+description: The '/sys/config/group-policy-application' endpoint is used to configure the global mode for group policy application.
+---
+
+# `/sys/config/group-policy-application`
+
+The `sys/config/group-policy-application` endpoint can be used to configure the
+mode of policy application for identity groups in Vault.
+
+Vault allows you to add entities and groups from any namespace into an identity group.
+However, historically, any policies attached to that group would only apply when the
+Vault token authorizing a request was created in the same namespace as that group,
+or a descendent namespace. This endpoint allows relaxing that restriction: when the mode is set to the default,
+`within_namespace_hierarchy`, the historical behaviour is maintained,
+but when set to `any`, group policies apply to all members of a group,
+regardless of what namespace the request token came from.
+
+Note that this configuration will be replicated between primary and secondaries, that
+is to say, primaries cannot have a different policy application mode to secondaries.
+
+## Get Group Policy Application Information
+
+~> **Enterprise Only** – This endpoint requires Vault Enterprise.
```

Comment:
Since the API is marked `ENT` in the side nav, would it make it more clear to just have this kind of warning once and at the top of the documentation, similar to how we have it here: https://developer.hashicorp.com/vault/docs/enterprise/performance-standby?

Reply:
I kept the docs consistent with how I saw other enterprise-only APIs documented - this is a non-API doc so I think that's why it's different!

Reply:
Oh, actually we have a slightly different approach where all of the APIs are Ent only. Updating! Thanks!